### ISSUE TITLE
Bug KillTimer - RoundStart

### ISSUE THREAD

**Issue by crashzk** [safra36/PlayerSkin#2]
I'm using plugin version 5.0.1 (Build 4):
```
[SM] Listing 42 plugins:
  01 "CS:GO PugSetup: RWS balancer" (2.0.5) by splewis
  02 "Basic Ban Commands" (1.9.0.6280) by AlliedModders LLC
  03 "Admin File Reader" (1.9.0.6280) by AlliedModders LLC
  04 "CS:GO PugSetup: autokicker" (2.0.5) by splewis
  05 "Sound Commands" (1.9.0.6280) by AlliedModders LLC
  06 "RankMe" (3.0.3.Kento.31.2) by lok1, Scooby, Kento, pracc, Kxnrl, CrazyHackGUT
  07 "CS:GO PugSetup: team names setter" (2.0.5) by splewis
  08 "[SWGM] Admin" (1.0) by Someone
  09 "CS:GO PugSetup: damage printer" (2.0.5) by splewis
  10 "Chat-Processor" (2.2.0) by Keith Warren (Drixevel)
  11 "Weapons & Knives" (1.3.3) by kgns | oyunhost.net
  12 "hextags" (1.45) by Hexah
  13 "REGEX word filter" (1.2) by Twilight Suzuka
  14 "Basic Commands" (1.9.0.6280) by AlliedModders LLC
  15 "Advanced Parachute" (1.0) by ESK0
  16 "Steam Works Group Manager" (1.7) by Someone
  17 "[CS:GO/CSS] AbNeR Round End Sounds" (4.0.1) by abnerfs
  18 "CS:GO PugSetup: hostname setter" (2.0.5) by splewis
  19 "Fun Commands" (1.9.0.6280) by AlliedModders LLC
  20 "Basic Chat" (1.9.0.6280) by AlliedModders LLC
  21 "Automatic Response" (1.1) by Plock - Edited: crashzk
  22 "Nextmap" (1.9.0.6280) by AlliedModders LLC
  23 "CS:GO PugSetup: team locker" (2.0.5) by splewis
  24 "Reserved Slots" (1.9.0.6280) by AlliedModders LLC
  25 "[CS:GO] Matchmaking Ranks by Points" (1.5) by Hallucinogenic Troll
  26 "Player Commands" (1.9.0.6280) by AlliedModders LLC
  27 "S-Downloader" (1.1.2.22 - Dev) by Starbish
  28 "ServerAdvertisements3" (3.1.3) by ESK0
  29 "Client Preferences" (1.9.0.6280) by AlliedModders LLC
  30 "Admin Menu" (1.9.0.6280) by AlliedModders LLC
  31 "Anti-Flood" (1.9.0.6280) by AlliedModders LLC
  32 "Gloves" (1.0.3) by kgns | oyunhost.net
  33 "Admin Help" (1.9.0.6280) by AlliedModders LLC
  34 "Fun Votes" (1.9.0.6280) by AlliedModders LLC
  35 "Basic Info Triggers" (1.9.0.6280) by AlliedModders LLC
  36 "Admin List" (1.2) by Fredd
  37 "Basic Comm Control" (1.9.0.6280) by AlliedModders LLC
  38 "[SWGM] Command Listener" (1.1) by Someone
  39 "CS:GO PugSetup: write team money to chat" (2.0.5) by Versatile_BFG/jkroepke
  40 "CS:GO PugSetup" (2.0.5) by splewis
  41 "Basic Votes" (1.9.0.6280) by AlliedModders LLC
  42 "PlayerSkin" (5.0.1 (Build 4)) by noBrain
```
However I get the following error in console:
```
L 06/20/2019 - 00:17:32: [SM] Exception reported: Invalid timer handle a740882 (error 1)
L 06/20/2019 - 00:17:32: [SM] Blaming: PlayerSkin.smx
L 06/20/2019 - 00:17:32: [SM] Call stack trace:
L 06/20/2019 - 00:17:32: [SM]   [0] KillTimer
L 06/20/2019 - 00:17:32: [SM]   [1] Line 89, E:\_MyFiles\Sourcemod\Files\Working On\Released\SkinShop\Version 5.0.1\addons\sourcemod\scripting\PlayerSkin.sp::RoundStart
```

**Comment by crashzk** [comment]
You have these errors too:
```
L 06/21/2019 - 10:43:01: [SM] Exception reported: Handle aa7508e1 is invalid (error 3)
L 06/21/2019 - 10:43:01: [SM] Blaming: PlayerSkin.smx
L 06/21/2019 - 10:43:01: [SM] Call stack trace:
L 06/21/2019 - 10:43:01: [SM]   [0] CloseHandle
L 06/21/2019 - 10:43:01: [SM]   [1] Line 701, E:\_MyFiles\Sourcemod\Files\Working On\Released\SkinShop\Version 5.0.1\addons\sourcemod\scripting\PlayerSkin.sp::SkinMenu
L 06/21/2019 - 10:43:05: [SM] Exception reported: Handle aa8f08e1 is invalid (error 3)
L 06/21/2019 - 10:43:05: [SM] Blaming: PlayerSkin.smx
L 06/21/2019 - 10:43:05: [SM] Call stack trace:
L 06/21/2019 - 10:43:05: [SM]   [0] CloseHandle
L 06/21/2019 - 10:43:05: [SM]   [1] Line 701, E:\_MyFiles\Sourcemod\Files\Working On\Released\SkinShop\Version 5.0.1\addons\sourcemod\scripting\PlayerSkin.sp::SkinMenu
```

**Comment by safra36** [comment]
I'm afraid it's too late to answer but the issue has been resolved, thanks for reporting
